modules/effects/src/effects_feature_module.ts

```diff
@@ -1,20 +1,36 @@
-import { NgModule, Inject, Optional } from '@angular/core';
-import { StoreRootModule, StoreFeatureModule } from '@ngrx/store';
+import { NgModule, Inject } from '@angular/core';
+import { Store } from '@ngrx/store';
 import { EffectsRootModule } from './effects_root_module';
 import { FEATURE_EFFECTS } from './tokens';
+import { getSourceForInstance } from './effects_metadata';
+
+export const FEATURE_EFFECTS_INIT = '@ngrx/feature-effects/init';
```

Comment:
I think the prefix should remain `@ngrx/effects/something`. What about `@ngrx/effects/(update-effects or update)`?

Reply:
Yea you're right.
My vote goes for `@ngrx/effects/update-effects` to keep things consistent with `@ngrx/store/update-reducers`.

Plus we wouldn't need to change the root effects:
> I also would want to change the root effects action name (this will be a breaking change) to make it more clear that it's initiated by the root effects: from @ngrx/effects/init to @ngrx/root-effects/init

Reply:
👍